garrysmod/lua/includes/util/color.lua

```diff
@@ -36,16 +36,27 @@ function IsColor( obj )
 end
 
 
+
 --[[---------------------------------------------------------
-	Returns color as a string
+	Returns color as a string ( unlinks necessity of
+	using tostring for functionality )
 -----------------------------------------------------------]]
-function COLOR:__tostring()
+function COLOR:ToString()
 	
 	return string.format( "%d %d %d %d", self.r, self.g, self.b, self.a )
 	
 end
 
 --[[---------------------------------------------------------
+	Returns color as a string
+-----------------------------------------------------------]]
+function COLOR:__tostring()

```

Comment:
(Actually nevermind - you'll want to leave `__tostring` to return only the numbers if you adapt `ToString` to output `Colour(R, G, B, A)` yes?)